specification/web/resource-manager/Microsoft.Web/stable/2022-03-01/AppServiceEnvironments.json

```diff
@@ -2293,6 +2435,54 @@
         }
       }
     },
+    "CustomDnsSuffixConfiguration": {
+      "description": "Full view of the custom domain suffix configuration for ASEv3.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "./CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "CustomDnsSuffixConfiguration resource specific properties",
+          "type": "object",
+          "properties": {
+            "provisioningState": {
+              "enum": [
+                "Succeeded",
+                "Failed",
+                "Degraded",
+                "InProgress"
+              ],
+              "type": "string",
+              "readOnly": true,
+              "x-ms-enum": {
+                "name": "CustomDnsSuffixProvisioningState",
+                "modelAsString": false
```

Comment:
Are you sure it is `false`?

Reply:
Yes, based on the considerations described [here](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/creating-swagger.md#modelAsString). the enum values listed here are unlikely to change, we have a friendly name for it, and we can include additional information in the provisioningDetails field as needed should the state be undescriptive enough alone.